src/zmq/zmqpublishnotifier.cpp

```diff
@@ -9,8 +9,13 @@
 
 static std::multimap<std::string, CZMQAbstractPublishNotifier*> mapPublishNotifiers;
 
+static const char *MSG_HASHBLOCK = "hashblock";
+static const char *MSG_HASHTX    = "hashtx";
+static const char *MSG_RAWBLOCK  = "rawblock";
+static const char *MSG_RAWTX     = "rawtx";
+
 // Internal function to send multipart message
-static int zmq_send_multipart(void *sock, const void* data, size_t size, ...)
+static int zmq_send_multipart_keepalive(void *sock, const void* data, size_t size, ...)
```

Comment:
I renamed it to `_keepalive` because it does not "close" the multipart message. All messaged are sent with `ZMQ_SNDMORE` and I wanted to increase the awareness of that by renaming the function.